src/librustc/middle/cstore.rs

```diff
@@ -52,7 +52,6 @@ pub use self::NativeLibraryKind::*;
 
 #[derive(Clone, Debug)]
 pub struct LinkMeta {
-    pub crate_name: Symbol,
     pub crate_hash: Svh,
 }
```

Comment:
This is one of the weirder structures in the whole compiler and it'd be nice if it'd just die already.